public/js/views/management.jsx

```diff
@@ -70,14 +54,9 @@ export default class Management extends Component {
     this.props.getUserSubscriptions();
   }
 
-  userSignIn = e => {
-    e.preventDefault();
-    this.props.userSignIn();
-  }
-
-  userSignOut = e => {
+  showSignOut = e => {
     e.preventDefault();
-    this.props.userSignOut();
+    this.props.showSignOut();
```

Comment:
I don't know how consistent we've been but I've been tending to call things like this handle[Whatever]. It would help to distinguish from the prop action with the same name.